src/bundle/Resources/views/parts/form/location_bookmark.html.twig

```diff
@@ -0,0 +1,18 @@
+{% trans_default_domain 'locationview' %}
+
+<div>
+    {% if form.vars.value.bookmarked %}
+        <div title = {{ 'location.bookmark.remove'|trans|desc('Remove from Bookmarks') }}>
+            <svg class="ez-icon">
+                <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#bookmark-active"></use>
+            </svg>
+        </div>
+    {% else %}
+        <div title = {{ 'location.bookmark.add'|trans|desc('Add to Bookmarks') }}>
```

Comment:
Do you need a `div` here?

Reply:
Can't add title to svg tag I believe.

Reply:
Anyway, @dew326 told me the HTML structure will be rewritten to implement an animation.